scenarios/hitech95/mail-generic-bf.yaml

```diff
@@ -0,0 +1,29 @@
+# Global brouteforce
+type: leaky
+#debug: true
+name: crowdsecurity/email-generic-bf
+description: "Detect generic email brute force"
+filter: "evt.Meta.log_type == 'mail_auth' && evt.Meta.sub_type == 'auth_fail'"
+groupby: evt.Meta.source_ip
+capacity: 5
+leakspeed: "10s"
+blackhole: 1m
+labels:
+ service: email
+ type: bf
+ remediation: true
+# Per user brouteforce
```

Comment:
Don't forget the separators `---` and rerun tests so it will detect user enum. Because in your tests there is only 2 generic-bf triggered.

Reply:
Got it!
I've f-pushed the changes 😃